flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/AbstractStreamOperator.java

```diff
@@ -208,13 +208,13 @@ public MetricGroup getMetricGroup() {
 	}
 
 	@Override
-	public final void initializeState(OperatorStateHandles stateHandles) throws Exception {
+	public final void initializeState(OperatorSubtaskState stateHandles) throws Exception {
 
 		Collection<KeyedStateHandle> keyedStateHandlesRaw = null;
 		Collection<OperatorStateHandle> operatorStateHandlesRaw = null;
 		Collection<OperatorStateHandle> operatorStateHandlesBackend = null;
 
-		boolean restoring = null != stateHandles;
+		boolean restoring = (null != stateHandles);
```

Comment:
+1 to keep the parenthesis

I think we should let contributors use such styles at their discretion